src/App.jsx

```diff
@@ -10,19 +11,38 @@ class App extends React.Component {
     selectedUserId: 0,
   };
 
+  componentDidMount() {
+    getTodos()
+      .then((todos) => {
+        this.setState({ todos });
+      });
+  }
+
+  selectUser = (userId) => {
+    this.setState({ selectedUserId: userId });
+  }
+
+  clearUser = () => {
+    this.setState({ selectedUserId: 0 });
+  }
+
   render() {
     const { todos, selectedUserId } = this.state;
 
     return (
       <div className="App">
         <div className="App__sidebar">
-          <TodoList todos={todos} />
+          <TodoList
+            todos={todos}
+            selectedUser={selectedUserId}
+            selectUser={this.selectUser}
+          />
         </div>
 
         <div className="App__content">
           <div className="App__content-container">
-            {selectedUserId ? (
-              <CurrentUser userId={selectedUserId} />
+            {selectedUserId && selectedUserId !== null ? (
```

Comment:
Too complicated logic to use in `render` method.
I would suggest creating class variable, smth like `shouldRenderCurrentUser` and use it here.